### ISSUE TITLE
Update 1378

### ISSUE THREAD

**Issue by bambooforest** [phoible/dev#318]
@cormacanderson notes that:

In GM 1378 (Mawa) the inventory given lists a number of phonemes, i.e. /ʔ/, /ʔw/, /h/, /f/, /z/ that the source (Roberts 2009) explicitly says are not phonemic.

Update.

**Comment by bambooforest** [comment]
@drammock -- Dr. Green was pretty clear about these:

https://github.com/phoible/dev/blob/master/raw-data/GM/gm-afr-inventories.tsv#L6703-L6707

but the original text argues that they are not phonemic (see attachment). 

<img width="444" alt="Screen Shot 2020-11-11 at 13 36 18" src="https://user-images.githubusercontent.com/1073582/98812515-f0a89480-2422-11eb-8e32-fc04269bf149.png">

**Comment by cormacanderson** [comment]
Part of the problem here is that the original is a prosodic account. If you have no way of listing the prosodies in the database, then you have no way of faithfully reflecting the source. One solution would be to try to deprosodicise it, so to speak, i.e. convert the prosodic account into a phonemic one. However, the prosodies are used in Chadic linguistics for pretty good reason, because it works very well for those languages. Conversion is not easy and in most cases will require a complete reanalysis with a very messy distributional statement, e.g. massive cooccurrence restrictions, minimal or no contrast in lots of positions. This is a problem throughout all of Chadic, especially the Central branch.

**Comment by bambooforest** [comment]
I made a PR here to remove these #319 @drammock

**Comment by drammock** [comment]
why does #319 remove `f h z` ?  The source doesn't say "they're not phonemes" it says "loanwords only", which we treat as phonemes but mark as "marginal" (which is how they currently are in the source):

https://github.com/phoible/dev/blob/9d21f8fa7d8bb592f6aa6378fb456757354d1441/raw-data/GM/gm-afr-inventories.tsv#L6703-L6707

**Comment by bambooforest** [comment]
Because my reading of that passage is that everything is phonetic.

**Comment by cormacanderson** [comment]
I would remark that, if you don't mark the prosodies, /ʔw/ is also phonemic, as the source considers it parallel to /ʔ/ initially when a word has w-prosody. This is a slippery slope though, as I point out in my comment above.

**Comment by drammock** [comment]
> Because my reading of that passage is that everything is phonetic.

I read it as "here are the reasons I don't include these segments: ʔ is phonetic, ʔw is not a phoneme for reasons I discuss later, and h f z are (technically phonemic/contrastive but) only present in loan words."  

> I would remark that, if you don't mark the prosodies, /ʔw/ is also phonemic, as the source considers it parallel to /ʔ/ initially when a word has w-prosody. This is a slippery slope though, as I point out in my comment above.

I'm not arguing with that; I know little about prosodic analysis of Chadic languages, so I'm happy to defer to your expertise here. I'm arguing that when the source says "occurs only in loanwords" we have an established way of handling that.

**Comment by cormacanderson** [comment]
It was more in the way of an "also", rather than a "but". I concur with your reading of the source. If you also consider loanwords then /f/, /h/, /z/ are phonemes. Without the prosodies, you probably have to consider /ʔw/ to be one too, although it's messy. /ʔ/ is phonetic in my view, although that only if you allow the phonology to "see" morphology. The situation he describes there sounds like that of German, where /ʔ/ is listed as a phoneme in UZ 2184. For the original source there:
<img width="835" alt="Kohler (1990)" src="https://user-images.githubusercontent.com/8975419/98846842-6f172d80-2447-11eb-83f6-f2a1f7dd3dba.png">

**Comment by drammock** [comment]
> I concur with your reading of the source. If you also consider loanwords then /f/, /h/, /z/ are phonemes.

OK, sounds like we agree then @cormacanderson. We typically include present-in-loanwords-only phonemes, but mark them as "marginal".

> Without the prosodies, you probably have to consider /ʔw/ to be one too

I have no problem with an author doing a phonological analysis that is morphology-aware, or prosody-aware. So I have no problem excluding /ʔw/ in this case.

**Comment by bambooforest** [comment]
I'm confused. Yes, we include the marginals (although my reading is that they could be phonetic, and as such are excluded from the chart).

But was is the status of /ʔw/ - @cormacanderson you write phonemic, @drammock you say no problem excluding it.

**Comment by cormacanderson** [comment]
In many Chadic languages, palatalisation and labialisation affect both consonants and vowels in a given domain. So you have a root like /tsar/ that can be palatalised (under some morphological operation) to [-tʃɛr-]. The descriptive practice would be to write this ʲ/tsar/ (this example is simplified but in the spirit of it) and they call the ʲ prosody, a suprasegmental feature that applies to a given domain. These don't just occur as a result of morphological operations: we could equally have a root  ʲ/tsar/.
To give a concrete example, in a language like Cuvok (PH 866) we have a vowel system /a ə/ and two prosodies of palatalisation and labialisation, with allophones as follows:
/a/ [a], ʲ/a/ [ɛ], ʷ/a/ [ɔ], ʲʷ/a/ [œ]
/ə/ [ə], ʲ/ə/ [i], ʷ/ə/ [u], ʲʷ/ə/ [y]
Basically, in Mawa, word initial /V-/ is pronounced with an initial glottal stop [ʔV-], while word initial ʷ/V-/, that is under the labialisation prosody, is pronounced [ʔw].
Really, it might be most in the spirit of the original sources to include these prosodies the same way that you include tone, as they are similarly suprasegmental. If you don't include them, that kind of amounts to a reanalysis, in which the likes of /ʔw/ have to be considered phonemic.
I've been going through these cases and how they are dealt with, not just in PHOIBLE but also elsewhere, also checking back to the original sources. I'll send you on what I come up with before your next update.

**Comment by drammock** [comment]
Wow, cool! Thanks for explaining the details. It sounds a lot like the retracted tongue root harmony in Mongolian.
